Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/model/view/FloatSliderSetting.kt

```diff
@@ -10,7 +10,11 @@ import java.math.BigDecimal
 import java.math.MathContext
 
 open class FloatSliderSetting : SliderSetting {
-    var floatSetting: AbstractFloatSetting
+    protected val floatSetting: AbstractFloatSetting
```

Comment:
Why wouldn't you just mark it as `private` like it is in IntSliderSetting?

Reply:
It's accessed by PercentSliderSetting.

Reply:
Oh sorry! Nevermind then